packages/shared/package.json

```
@@ -5,9 +5,10 @@
   "type": "module",
   "private": true,
   "exports": {
+    "./db/*": "./src/db/*.ts",
     "./knotenpunkt/*": "./src/knotenpunkt/*.ts",
     "./s3/*": "./src/s3/*.ts",
-    "./db/*": "./src/db/*.ts",
+    "./services/*": "./src/services/*.ts",
     "./types/*": "./src/types/*.ts"
   },
```

The shared package is only for our internal structuring. I would not mind exporting everything from the package. e.g. like this:
```suggestion
"exports": {
  "./*": "./*"
},
```

---

Good point. I will export the whole ./src/* folder.